src/codeu/chat/common/NetworkCode.java

```diff
@@ -36,4 +36,6 @@
       RELAY_READ_RESPONSE = 28,
       RELAY_WRITE_REQUEST = 29,
       RELAY_WRITE_RESPONSE = 30;
+  	  SERVER_INFO_REQUEST = 31;
```

Comment:
Oh, good point- you're absolutely correct; these are not the end of lines. I'll make that change and push again